packages/openapi-react-query/package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "openapi-react-query",
+  "description": "Fast, type-safe @tanstack/react-query client to work with your OpenAPI schema.",
+  "version": "0.0.0",
+  "author": {
+    "name": "Martin Paucot",
+    "email": "contact@martin-paucot.fr"
+  },
+  "license": "MIT",
+  "type": "module",
+  "main": "./dist/index.js",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      },
+      "require": {
+        "types": "./dist/cjs/index.d.cts",
+        "default": "./dist/cjs/index.cjs"
+      }
+    },
+    "./*": "./*"
+  },
+  "homepage": "https://openapi-ts.dev",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/openapi-ts/openapi-typescript",
+    "directory": "packages/openapi-react-query"
+  },
+  "bugs": {
+    "url": "https://github.com/openapi-ts/openapi-typescript/issues"
+  },
+  "keywords": [
+    "openapi",
+    "swagger",
+    "rest",
+    "api",
+    "oapi_3",
+    "oapi_3_1",
+    "typescript",
+    "fetch",
+    "react",
+    "react-query",
+    "tanstack"
+  ],
+  "scripts": {
+    "build": "pnpm run build:clean && pnpm run build:esm && pnpm run build:cjs",
+    "build:clean": "del-cli dist",
+    "build:esm": "tsc -p tsconfig.build.json",
+    "build:cjs": "esbuild --bundle --platform=node --target=es2019 --outfile=dist/index.cjs --external:@redocly/ajv --external:@redocly/openapi-core --external:typescript src/index.ts",
```

Comment:
👍 This looks correct at first glance

Reply:
Double check the index.cjs output directory. I think it doesn't match the expected path set in the exports property.

Reply:
Good point !